src/containers/ActivityHistoryTable/index.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import React, { useState } from 'react';
+import Image from 'next/image';
+
+import CCard from 'src/components/CCard';
+import CStreamStatus, { StreamStatus } from 'src/components/CStreamStatus';
+import CStreamType from 'src/components/CStreamType';
+
+import { clipText } from 'src/utils/clipText';
+import { formatNumber } from 'src/utils/formatNumber';
+
+import usdt from 'public/images/usdt.svg';
+import divider from 'public/images/divider.svg';
+
+import Funnel from 'src/svgs/Funnel';
+import MagnifyingGlass from 'src/svgs/MagnifyingGlass';
+
+import * as Styled from './styles';
+import useFetchHistory from './useFetchHistory';
+
+const ActivityHistoryTable = () => {
+  const [selectedStatus, setSelectedStatus] = useState<StreamStatus | ''>(
+    StreamStatus.ONGOING
+  );
+
+  const handleStreamStatusChange = (value: StreamStatus) => {
```

Comment:
Remove the function and pass `setSelectedStatus` directly to the onChange props.